hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcClient.java

```diff
@@ -134,6 +136,7 @@
 
   /** Fairness manager to control handlers assigned per NS. */
   private RouterRpcFairnessPolicyController routerRpcFairnessPolicyController;
+  private Map<String, AtomicLong> rejectedPermitsPerNs = new ConcurrentHashMap<>();
```

Comment:
For better performance, can we replace AtomicLong with LongAdder. Just a suggestion.

Reply:
Updated.